apps/src/lib/ui/SchoolInfoInterstitial.jsx

```diff
@@ -217,7 +217,7 @@ export default class SchoolInfoInterstitial extends React.Component {
     if (this.isBlank(country)) {
       errors.country = true;
       isValid = false;
-    } else {
+    } else if (country === 'United States') {
```

Comment:
nit: you can remove the `if (country === 'United States')` line on line 225.

Reply:
Thanks!